integration-tests/utils/catalog-utils.js

```diff
@@ -17,7 +17,9 @@ const linkDataToWorkspace = async (page, testUrl, token) => {
   await click(page, checkbox({ text: 'Granted', isDescendant: true }))
   // TODO: add test data with granted access DC-321
   await clickTableCell(page, { tableName: 'dataset list', columnHeader: 'Dataset Name', text: 'Readable Catalog Snapshot 1', isDescendant: true })
-  await noSpinnersAfter(page, { action: () => click(page, clickable({ textContains: 'Link to a workspace' })) })
+  await click(page, clickable({ textContains: 'Prepare for analysis' }))
+  await delay(20000)
```

Comment:
Why use a delay instead of `noSpinnersAfter`?

Reply:
I was finding with the polling it wasn't picking up the spinners with `noSpinnersAfter`. If it should be, then I can look into it a bit more.